### ISSUE TITLE
Incorrect reporting of year if publication date does not match publication year

### ISSUE THREAD

**Issue by episanty** [Langenscheiss/bibitnow#37]
Consider the paper https://doi.org/10.1038/nphoton.2014.293

This was published online on 8 December 2014, but published in print on the January 2015 issue, so the formal publication year is 2015. However, the bibtex produced by BibItNow attributes the year as 2014.

**Comment by Langenscheiss** [comment]
Hi. 
Thanks for reporting. This has zero to do with years not matching dates. It's just the way Nature provides publication date data. I have refined the site adjuster to take that into account. The next version should be online soon. 

I have spent a lot of time and effort on adding the publication address field, but the problem is that there are so many publisher imprints nowadays that this takes a lot of time. I will keep on improving the address feature for the next builds and gonna push the current one since it has been a while. There is also crossref support planned in the future. That should further enhance data reliability.
